command/command.ts

```diff
@@ -1712,7 +1716,14 @@ export class Command<
 
       if (this._useRawArgs) {
         await this.parseEnvVars(ctx, this.envVars);
-        return await this.execute(ctx.env, ctx.unknown);
+
+        if (ctx.execute) {
+          return await this.execute(ctx.env, ctx.unknown);
+        } else {
+          // TODO: Resolve typing for options
```

Comment:
Not sure how to satisfy `options` here since it's normally parsed later with more context.